lib/std/coff.zig

```diff
@@ -162,20 +163,51 @@ pub const Coff = struct {
         try self.loadOptionalHeader();
     }
 
+    fn readStringFromTable(self: *Coff, offset: u32, buff: []u8) !u32 {
+        if (self.coff_header.pointer_to_symbol_table == 0) {
+            // No symbol table therefore no string table
+            return error.MissingStringTable;
+        }
+        const string_table_offset = self.coff_header.pointer_to_symbol_table + (self.coff_header.number_of_symbols * 18) + offset;
```

Comment:
Please add a comment explaining what `18` is.

Reply:
Will do